src/store.ts

```diff
@@ -613,6 +618,16 @@ export default function Store(
         );
     };
 
+    this.getUserAttributes = (mpid: MPID): UserAttributes =>
+        this._getFromPersistence(mpid, 'ua') || {};
```

Comment:
You should also pass the <type> like line 542-543 above.

Reply:
That's a good call out. I would like to address it as part of a separate cleanup PR since I think we missed a few of these during the individual PRs.